### PR TITLE
Added mouse_down(), mouse_is_pressed and mouse_up docs

### LINKED ISSUE #82: As a student, I want docs for mouse_down() and mouse_up()
Currently `key_pressed()` is taught in Session 6 as a method for a run-once function. `mouse_down()` exists in our source code but we have no docs for it presently.

AC:
* [x] Add docs for `mouse_down` and explain that it's useful for running code just once when being clicked (compared to `mouse_is_pressed` in a `draw()` loop)
* [x] Add docs for `mouse_up`

### PR DESCRIPTION
**Pull request by Descent098**

### CODE REVIEW

**Comment by Descent098** [comment]
resolves #82

File: docs/input.md [hunk 1]
```diff
@@ -108,6 +108,115 @@ which results in:
...
 
+
+### Check for mouse presses
+
+Spark features multiple ways to check for user mouse presses:
+
+| Name                                  | Type                  | Description                                                            |
+|---------------------------------------|-----------------------|------------------------------------------------------------------------|
+| [mouse_is_pressed](#mouse_is_pressed) | Variable              | A boolean variable that is True when a mouse button is pressed         |
+| [mouse_down()](#mouse_down)           | User defined function | A definable function that activates when a mousebutton is pressed down |
```
Comment (AlphaRLee) [inline docs/input.md]: Consistency: Use `mouse button` instead of `mousebutton`

Comment (Descent098) [reply docs/input.md]: Resolved in latest commit

File: docs/input.md [hunk 2]
```diff
@@ -108,6 +108,115 @@ which results in:
...
+*Print "Mouse Button Pressed Down" if a mouse button is pressed down*
+
+```python hl_lines="9 10"
+%%ignite
+
+def setup():
+    size(200, 200)
+
+def draw():
+    ... # Does nothing
```
Comment (AlphaRLee) [inline docs/input.md:175]: Try using something with minimal effects like `background("white")` (or possibly `pass`, but we haven't taught that) in the docs. These examples should be fully functional or be obvious as to why they don't work.
```suggestion
    background("white")
```

File: docs/input.md [hunk 3]
```diff
@@ -108,6 +108,115 @@ which results in:
...
+Results in:
+
+![mouse_down() example](img/mouse_down.png){: loading=lazy }
+
+
+#### mouse_up()
+
+```python
+def mouse_up():
+    # Your code goes here
```
Comment (AlphaRLee) [inline docs/input.md:190]: Same as comment above, add something in here so it's a functional example
```suggestion
    print("Mouse released")
```

File: docs/input.md [hunk 4]
```diff
@@ -108,6 +108,115 @@ which results in:
...
+*Print "Mouse Button Released" if a mouse button is released after being pressed down*
+
+```python hl_lines="9 10"
+%%ignite
+
+def setup():
+    size(200, 200)
+
+def draw():
+    ... # Does nothing
```
Comment (AlphaRLee) [inline docs/input.md:210]: Same as comment above
```suggestion
    background("white")
```

**Review by AlphaRLee — COMMENTED** [review]
Minor corrections needed to have non-empty examples, then it should be good to merge

**Comment by Descent098** [comment]
> Minor corrections needed to have non-empty examples, then it should be good to merge

So in examples of user definable functions draw() intentionally utilizes pythons ellipse class. Since draw() literally is only being used to run perpetually, these examples are written this way to show that **only** the functions are doing something, not draw(). All the examples are copy-paste runable

**Review by AlphaRLee — APPROVED** [review]

**Pull request merged**